packages/eslint-plugin-angular-template-spacing/src/collection.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "../../../node_modules/@angular-devkit/schematics/collection-schema.json",
```

Comment:
I'm not sure if we can rely on that path. This will work in our monorepo, but not outside of it.

Reply:
![Screen Shot 2022-09-15 at 12 55 15](https://user-images.githubusercontent.com/19550608/190386515-b70d5b78-d638-4b3d-9506-d26c1cb3d81d.png)
That's what ngrx has in their repo. Maybe we can somehow reuse that? But I don't know about the difference between `@angular-devkit/schematics` and `@schematics/angular`.

Reply:
I don't think we want to extend angulars schematics. However the `$schema` key is just for schema validation, i pointed it directly to git url.